qemu/tests/cfg/numa_cpu.cfg

```diff
@@ -0,0 +1,77 @@
+- numa_cpu:
+    type = numa_cpu
+    kill_vm = yes
+    no Host_RHEL.m6, Host_RHEL.m7
+    # '-numa cpu' was added from QEMU 3.0(av-8.0),rhel-8.2 don't support it.
+    # 'dies' was added from QEMU 4.1(av-8.1, rhel-8.3), rhel-8.2 don't support.
+    # libvirt still does not support it by 8.3, so test it from 8.3
+    no Host_RHEL.m8.u0, Host_RHEL.m8.u1, Host_RHEL.m8.u2
+    mem = 4096
+    ppc64, ppc64le:
+        smp =2
+        vcpu_sockets = 1
+        vcpu_dies = 1
```

Comment:
With default `vcpu_dies = INVALID` in base.cfg, vt handle it as `1` finally:
https://github.com/avocado-framework/avocado-vt/blob/9c82462cbaa394be237e71adaae711ec0222097e/virttest/qemu_vm.py#L1523

That is the reason that i set it to `1` here.
We can also leave it, and handle `INVALID` in this case with the same way as vt, what do you think ?

Reply:
@yanan-fu Let me clarify, when `vcpu_dies = INVALID`, `enable_dies` is `False`, this propoty will not be set on `-smp` command line. But if `vcpu_dies = 1`, it will.
https://github.com/avocado-framework/avocado-vt/blob/9c82462cbaa394be237e71adaae711ec0222097e/virttest/qemu_vm.py#L1606-L1607

Reply:
Yes, it see that.  I mean it use `dies=1` to compute the cpu topology.
Is it must to remove dies from cli for ppc ?   If so, we can not use `vcpu_dies=1` in this case, i will update it.

Reply:
Yes, PPC does not support this property and will not apply it if you force to set it.

Reply:
@PaulYuuu  Updated, could you help review again, thanks!